src/ui/mainwindow.cc

```diff
@@ -748,7 +747,21 @@ MainWindow::MainWindow( Config::Class & cfg_ ):
   if ( cfg.mainWindowGeometry.size() )
     restoreGeometry( cfg.mainWindowGeometry );
 
+#if defined( Q_OS_LINUX )
```

Comment:
other parts use `#if !defined( Q_OS_WIN )`, unified with the same condition?

Reply:
Only Linux needs `defaultInterfaceStyle` because everyone has a different `defaultInterfaceStyle` :sweat_smile:

On macOS, the default would be forced to use Fusion.